social/pushbullet/57-pushbullet.js

```diff
@@ -511,14 +511,24 @@ module.exports = function(RED) {
                     res.send("[]");
                     return onError(err, config);
                 }
-                res.send(JSON.stringify(chans.devices));
+                res.send(JSON.stringify(filterdActiveDevices(chans.devices)));
             });
         }
         else {
             res.send("[]");
         }
     });
 
+    function filterdActiveDevices(devices){
+        var activeDevices = [];
+        for(var i=0;i<devices.length;i++){
+            if(chans.devices[i].active){
+                activeDevices.push(devices[i]);
+            };

```

Comment:
sadly we have the jsHint rules set fairly high so this ; triggers it. It can be removed.

Reply:
sorry.
fix it.

Reply:
no problem - and you also need to accept/sign the CLA. Thanks